etc/formatter/Dockerfile

```diff
@@ -13,6 +13,4 @@ RUN npm install -g prettier@2.2.1
 
 # TODO:
 # In later iterations, we can also install all the Rustup cargofmt stuff here
-# as well as Pants to perform containerized linting.
-
-USER nobody
```

Comment:
this is the main change here. rest is just Prettier working its magic.

Reply:
Looks like this will cause it to run as root. Would that make the files be owned as root after running the formatter? Maybe not since all files changes already exist, so maybe ownership is retained.

However, I'd like if we ran the container as the current uid:gid from the host. The Makefile already sets UID and GID as env vars, so in the docker-compose you should be able to set `user: "${UID}:${GID}"`. How does that sound?

Reply:
Done, good idea